sw/device/silicon_creator/lib/epmp_state.c

```diff
@@ -4,8 +4,11 @@
 
 #include "sw/device/silicon_creator/lib/epmp_state.h"
 
+#include "sw/device/lib/base/bitfield.h"
 #include "sw/device/lib/base/csr.h"
 #include "sw/device/lib/base/hardened.h"
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/silicon_creator/lib/rom_print.h"
```

Comment:
Would it make sense to move this into a separate module? `epmp_debug.c`? Even if it gets optimized out, it may make sense for the developers to have to explicitly add this dependency.

Reply:
Done.